scripts/configure-multi-node-aio.yml

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
There's a lot being done in https://github.com/rcbops/rpc-openstack/blob/master/scripts/bootstrap-aio.yml which is not being done here, and also a lot being done by the bootstrap is unnecessary (configuring loopback disks and such). Perhaps we could figure out a way to re-use the same thing and enable/disable actions based on the environment being deployed? Either that or we should differentiate this one properly.

It seems that the MNAIO tooling does a ton of config related things - why're we using the AIO bootsrap?

I would ideally prefer to use the same config prep mechanism for both test environments. It makes it easier to context switch.